linera-views/src/views/key_value_store_view.rs

```diff
@@ -40,6 +40,106 @@ static KEY_VALUE_STORE_VIEW_HASH_RUNTIME: LazyLock<HistogramVec> = LazyLock::new
     .expect("Histogram can be created")
 });
 
+#[cfg(with_metrics)]
+/// The runtime of get
+static KEY_VALUE_STORE_VIEW_GET_RUNTIME: LazyLock<HistogramVec> = LazyLock::new(|| {
```

Comment:
I believe we're using `LATENCY` instead of `RUNTIME` in the rest of the code for API metrics